core/templates/dev/head/pages/exploration_editor/statistics_tab/issues_directive.html

```diff
@@ -0,0 +1,28 @@
+<div class="oppia-issues-issue-dir">
+  <div class="row oppia-issues-content oppia-issues-row-eq-height">
+    <div class="col-md-2 oppia-issues-border-gray oppia-ft-14">
+      <div ng-repeat="issue in issues" class="row">
+        <div ng-if="getIssueIndex(issue) == 0" class="issueNav text-center col-md oppia-issues-bor-btm oppia-issues-clr-green oppia-pt2 oppia-pb2 oppia-issues-bg-clr oppia-issues-text-white" ng-click="makeVisible(getIssueIndex(issue))" id="issueNav<[getIssueIndex(issue)]>">
```

Comment:
Please add helper functions to $scope to make it clearer what these conditions are validating.

Reply:
Done.

Reply:
Done.